frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/components/SearchApplication.js

```
@@ -68,213 +351,302 @@ const PTSearchApplication = ({tenantId, isLoading, t, onSubmit, data, count, set
+                        width: "100%",
+                        display: "flex", justifyContent: "space-between"
+                    }}>
+                        <SearchField>
+                            <label style={{
+                                fontFamily: "Poppins",
+                                fontWeight: 400,
+                                fontSize: "14px",
+                                lineHeight: "22px",
+                                letterSpacing: "0%",
+                                color: "#282828"
+                            }}>{t("PT_APPLICATION_NO_LABEL")}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)